calendar/calendar.yaml

```diff
@@ -377,3 +377,20 @@
       Notes & Agenda: https://docs.google.com/document/d/1GHi-NFHmDA2TnH1pDrs4O7OyIQdZRtKatGAdedxte4w
       Zoom: Provided in meeting notes
   organizer: woop
+
+- id: kf034
```

Comment:
Why not create your own calendar? Everything should be free.
e.g. people are already moving away from centralized calendar: https://github.com/kubeflow/community/pull/463

Reply:
@Bobgy, we don't really have a clear place to put it right now, and this will help us get the working group going